mobile_payment/apps.py

```diff
@@ -0,0 +1,50 @@
+from django.apps import AppConfig
+
+MODULE_NAME = "mobile_payment"
+
+DEFAULT_CFG = {
+      
+      "default_validations_disabled":False,
+      "gql_query_transactions_perms":["111001"],
+      "gql_query_payment_service_provider":   ["101305"],
+      "gql_mutation_create_transaction_perms":["1110002"],
+      "gql_mutation_update_transaction_perms":["1110004"],
+      "gql_mutation_verify_insuree_perms":["1110003"],
+      "gql_mutation_process_payment_perms":["1110001"],
+      "gql_mutation_create_paymnet_service_provider":["101306"],
+      "gql_mutation_update_paymnet_service_provider":["101307"],
+      "gql_mutation_delete_paymnet_service_provider":["101308"],
+        
```

Comment:
```suggestion
    "default_validations_disabled": False,
    "gql_query_transactions_perms": ["111001"],
    "gql_query_payment_service_provider": ["101305"],
    "gql_mutation_create_transaction_perms": ["1110002"],
    "gql_mutation_update_transaction_perms": ["1110004"],
    "gql_mutation_verify_insuree_perms": ["1110003"],
    "gql_mutation_process_payment_perms": ["1110001"],
    "gql_mutation_create_paymnet_service_provider": ["101306"],
    "gql_mutation_update_paymnet_service_provider": ["101307"],
    "gql_mutation_delete_paymnet_service_provider": ["101308"],
```

could you follow a consistent code style please? especially regarding indentation that is important in Python.

FYI, There are a few tools that allows you to automate the formatting.

Reply:
Noted all indentation practices will be followed and applied